projects/fsastorefrontstyles/scss/overrides/order-history.scss

```diff
@@ -1,22 +1,32 @@
 @import '../theme/fs/variables';
 
-%cx-order-history {
-  .cx-order-history-header {
-    @include headline-style;
-    padding: 1rem 0;
-    margin: 1rem auto 3rem;
-    text-align: center;
-  }
+%cx-fs-order-history {
+  @extend %cx-order-history;
```

Comment:
Because of different tag selector we need to extend the Spartacus' to get their styling.